src/helpers.tsx

```diff
@@ -2,28 +2,47 @@
 import { WrapperSettings } from '@storybook/addons';
 import { DEFAULT_PADDING } from './constants';
 
-type Option = {
-  name: string;
-  value: string;
-  default?: boolean;
+type Size = { name: string; value: string };
+
+type Option = Size & {default?: boolean}
+
+type Sizes = { [key: string]: Size };
+
+type PaddingConfig = {
+  sizes: Sizes;
+  defaultPadding: string;
```

Comment:
I think we should follow the new Backgrounds addon API:

https://github.com/storybookjs/storybook/blob/badece61154f4cd372404786186125a7369e362f/addons/backgrounds/src/containers/BackgroundSelector.tsx#L35-L39

Reply:
Done: